src/query/service/src/pipelines/processors/transforms/window/window_function.rs

```diff
@@ -106,6 +114,8 @@ pub enum WindowFunctionImpl {
     PercentRank,
     Lag(WindowFuncLagLeadImpl),
     Lead(WindowFuncLagLeadImpl),
+    FirstValue(WindowFuncFirstLastImpl),
+    LastValue(WindowFuncFirstLastImpl),
```

Comment:
I think we don't need make a `XXXImpl` for `first_value` and `last_value`. I will give a detail explanation below.